spec/controllers/taxons_controller_spec.rb

```diff
@@ -120,6 +123,16 @@
 
       expect(flash.now[:danger]).to match(/<a href="(.+)">taxon<\/a> with this slug already exists/)
     end
+
+    it "sends additional publish request to Publishing API for the Brexit taxon with 'cy' locale" do
+      build(:taxon, publication_state: "unpublished", content_id: brexit_taxon_content_id)
+      stub_any_publishing_api_publish
+
+      post :publish, params: { taxon_id: brexit_taxon_content_id }
+
+      assert_publishing_api_publish(brexit_taxon_content_id, update_type: nil)
```

Comment:
One more minor thing: this line would benefit from `locale: "en"`, as I think otherwise it's superseded by the next line.

Reply:
Ah so the request doesn't have `locale: "en"` in the body as it's optional and defaults to `en` in the Publishing API anyway (see https://github.com/alphagov/publishing-api/blob/master/doc/api.md#post-v2contentcontent_idpublish).

Reply:
Thanks :-).